ci/base.yml

```diff
@@ -15,11 +15,15 @@ repos:
   meta-qcom:
     url: https://github.com/Linaro/meta-qcom
 
-  poky:
-    url: https://git.yoctoproject.org/git/poky
+  oe-core:
```

Comment:
this name is hardcoded in some of our ci scripts (git grep poky -- ci/) and will break them.